.github/workflows/test_pr.yml

```diff
@@ -34,4 +34,50 @@ jobs:
         with:
           args: deploy -P roster-pr --only hosting:project-dashboard-pull-request
         env:
-          FIREBASE_TOKEN: ${{ secrets.FIREBASE_TOKEN }}
\ No newline at end of file
+        FIREBASE_TOKEN: ${{ secrets.FIREBASE_TOKEN }}
+
+
+# Only run the coverage once
+      name: run Coverage once
+      run: |
+        # var SUMMARY = [
+        #   '',
+        #   '=============================== Coverage summary ===============================',
+        #   'Statements   : 32.5% ( 39/120 )',
+        #   'Branches     : 38.89% ( 21/54 )',
+        #   'Functions    : 21.74% ( 5/23 )',
+        #   'Lines        : 31.93% ( 38/119 )',
+        #   '================================================================================',
+        #   ''
+        # ];
+        # SUMMARY = SUMMARY.split('\n')[5]; // 'Lines        : 31.93% ( 38/119 )'
+        # SUMMARY = SUMMARY.split(':')[1].split('(')[0].trim(); // '31.93%'
+        SUMMARY="$(react-scripts test "--coverageReporters=text-summary")"
+        TOKENS=($SUMMARY)
```

Comment:
Try to run commands in bash first to check if they are resulting the wanted output.
They summary will contain after that something like `> project-dashboard@0.1.0 test /home/mikko/workspace/project-dashboard > react-scripts test` which doesn't look correct.

Reply:
consider using `PERCENTAGE=$(npm test -- --coverage --coverageReporter='text-summary' | grep 'All files' | cut -d '|' -f 5)` as shown in the meeting. Other approaches are fine also as long as they work.